docs/UserGuide.md

```diff
@@ -403,3 +421,22 @@ The graphical user interface (GUI) is a form of user interface that allows users
+| `DESCRIPTION`                            | 1. This field can only contain alphanumeric characters, or punctuations (other than `/`), or spaces.<br/>2. The first character must be a letter or digit.<br/>3. This field is case-insensitive.                       | 1 - 255      | [`add`](#31-adding-a-task-add) [`find`](#35-locating-a-task-find) [`edit`](#38-editing-a-task-edit)                                                          |
+| `DEADLINE`                               | 1. This field should contain a valid date.<br/>2. This field should be in the format of `YYYY-MM-DD`.                                                                                                                   | NA           | [`add`](#31-adding-a-task-add) [`edit`](#38-editing-a-task-edit) [`sort`](#39-sorting-tasks-sort)                                                            |
+| `PRIORITY`                               | 1. This field should be in the format `low`, `medium`, or `high`.<br/>2. This field is case-insensitive.                                                                                                                | NA           | [`add`](#31-adding-a-task-add) [`find`](#35-locating-a-task-find) [`edit`](#38-editing-a-task-edit) [`sort`](#39-sorting-tasks-sort)                         |
+| `TAG`                                    | 1. This field can only contain alphanumeric characters.<br/>2. This field is case-insensitive.                                                                                                                          | 1 - 50       | [`add`](#31-adding-a-task-add) [`find`](#35-locating-a-task-find) [`edit`](#38-editing-a-task-edit)                                                          |
+| `INDEX`                                  | 1. This field should be a positive integer.<br/>2. The value of this field should not be more than the number of tasks displayed.                                                                                       | NA           | [`delete`](#34-deleting-a-task-delete) [`mark`](#36-marking-as-complete-mark) [`unmark`](#37-marking-as-incomplete-unmark) [`edit`](#38-editing-a-task-edit) |
+| `NAME_KEYWORD`<br/>`DESCRIPTION_KEYWORD` | 1. These fields should only contain alphanumeric characters.<br/>2. These fields should not contain any space in the middle.<br/>3. These fields are case-insensitive.                                                  | 1 - 63       | [`find`](#35-locating-a-task-find)                                                                                                                           |
```

Comment:
If we mention that the fields should only contain alphanumeric characters, do we still need to mention that the fields should not contain spaces?

Reply:
ohh okie thanks for pointing out